tests/whitebox/unittest/test_error_fmt.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2019 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Test error type string formatting.
+"""
+
+import argparse
+import unittest
+
+from stratis_cli._errors import StratisCliError
+from stratis_cli._errors import StratisCliRuntimeError
+from stratis_cli._errors import StratisCliPropertyNotFoundError
+from stratis_cli._errors import StratisCliPartialChangeError
+from stratis_cli._errors import StratisCliNoChangeError
+from stratis_cli._errors import StratisCliIncoherenceError
+from stratis_cli._errors import StratisCliInUseError
+from stratis_cli._errors import StratisCliUnknownInterfaceError
+from stratis_cli._errors import StratisCliEngineError
+from stratis_cli._errors import StratisCliActionError
+from stratis_cli._errors import StratisCliGenerationError
+from stratis_cli._errors import StratisCliEnvironmentError
+
+from stratis_cli._stratisd_constants import BlockDevTiers
+
+
+class ErrorFmtTestCase(unittest.TestCase):
+    """
+    Test stringification of various error types.
+    """
+
+    def StringNotEmpty(self, exception):
```

Comment:
This method does not need to be public so I'd name this `_string_not_empty`. Other than that, this method looks good.